<launch>
    <node name="geometric_verifier" pkg="geometric_verification" type="geometric_verification" output="screen">
	<param name="base_link_frame_id" value="odom_combined"/>
	<param name="sensor_frame_id" value="camera_0"/>
	<param name="window_size_in_frames" value="150"/>
	<param name="allowed_trans_x" value="2.0"/>
	<param name="allowed_trans_y" value="2.0"/>
	<param name="allowed_trans_z" value="2.0"/>
    </node>
</launch>

